benchexec/tools/theta.py

```diff
@@ -30,14 +31,24 @@ def project_url(self):
     def version(self, executable):
         return self._version_from_tool(executable)
 
+    def min_version(self, current, limit):
+        for i in range(len(current)):
+            if int(current[i]) < int(limit[i]):
+                return False
+        return True
```

Comment:
I think if you just use `[int(x) for x in current] >= limit` it does the same and works even if `limit` is a prefix of `current`, for which the current version crashes.